rpm/rpm.go

```diff
@@ -310,9 +310,15 @@ func CombinedProber(
 				// https://github.com/golang/go/blob/7ca6902c171b336d98adbb103d701a013229c806/src/net/http/transport.go#L74
 				transport.TLSClientConfig.KeyLogWriter = keyLogger
 			}
-			transport.TLSClientConfig.InsecureSkipVerify = true
 
-			foreignProbeClient := &http.Client{Transport: &transport}
+			transport.TLSClientConfig.InsecureSkipVerify = foreignProbeConfiguration.InsecureSkipVerify
+
+			if err := utilities.OverrideHostTransport(transport, foreignProbeConfiguration.ConnectToAddr); err != nil {
+				fmt.Printf("Fatal: Could not configure Transport: %s", err)
+				os.Exit(1)
```

Comment:
I would prefer that these errors bubble up through the normal paths and that we let the caller decide what to do in the case of an error. Does that seem unreasonable?

Reply:
I would too prefer that, but there's not a clear error exit point here.

Can you give guidance on your preference here? An error chan that's listened for?

Reply:
Okay. I have thought lots about this and here's what I think we can easily do ... 

Let's commit this now and then in the next few days I will go through the code and build a centralized error-logging channel that we can distribute to the functions that could generate errors in certain conditions. If those functions generate non-critical errors (I would consider the above one such example because being unable to launch a *single* probe should not stop the entire test [although it would be different if we saw the error persistently, but that's an issue for the thing listening to the error channel (and now there are *so* many nested brackets [because of asides] that I forgot my train of thought)]) then they log errors to that channel. The receiver for messages on that channel can then decide how the overall test should proceed. 

Besides the strained English in this message, does that plan sound reasonable?